Tasks/DownloadBuildArtifactsV0/main.ts

```diff
@@ -389,6 +414,36 @@ function configureDownloaderOptions(): engine.ArtifactEngineOptions {
     return downloaderOptions;
 }
 
+function extractTars(downloadPath: string, tarArchivesPaths: string[]): void {
+    const extractedTarsPath: string = path.join(tl.getVariable('Agent.TempDirectory'), 'extracted_tars');
```

Comment:
Please move 'extracted_tars' to constants

Reply:
Done